### ISSUE TITLE
Improve UI and UX

### ISSUE THREAD

**Issue by yogesh0509** [yogesh0509/web3-DREAM11#11]
- [ ] Improve component designs. 
- [ ] Migrate wagmi version v1 to v2.

**Comment by sanaa9012** [comment]
Hello, I would like to contribute in this issue. Can you assign it to me?

**Comment by yogesh0509** [comment]
I can only assign issues after 25th September but meanwhile can you share any design components or your approach to this issue?

**Comment by sanaa9012** [comment]
My approach for a project like web3-DREAM11 would be, the UI prioritizes clarity, ease of use, and interactivity for an immersive fantasy sports experience.

Key features include:

1. Home/Dashboard: Displays account details, tokens, leaderboard, live auctions, and notifications.
2. Team Selection & Auction: Player cards with stats, real-time auction status, bidding options, and token visibility.
3. Game Progress & Results: Real-time match stats, oracles integration for scores, and match results.
4. Profile/Wallet: Tracks player history, token balances, and transactions.
5. Blockchain Transparency: Smart contract viewer, transaction history, and data trust indicators.
6. Game Winner Determination: Shows rankings and scores verified by blockchain.
7. Onboarding: Interactive tutorials and guides.
Design focuses on mobile-first optimization, real-time updates, and seamless blockchain integration using React/Next.js, Web3.js, and Socket.io. This UI simplifies fantasy team management, auctions, and ensures transparency.

![a-minimal-ui-design-for-a-web3-fantasy-sports-plat-m19BHbvARAyAjgkrT8a4sw-Q0uTNeYjQ-ikvLAesJCexw](https://github.com/user-attachments/assets/9d470647-07e4-4ca1-8d7c-086715b1862d)
![a-minimalistic-web3-dream11-fantasy-sports-platfor-jR3JtnmBSwaF4Shg0d6qVw-Q0uTNeYjQ-ikvLAesJCexw](https://github.com/user-attachments/assets/6f5e71c6-3b93-4c9e-9bbe-c25e651c9d00)
![a-minimal-ui-with-the-following-features-1-home-da-jOW-bfFXSzKuih4hOs6x7A-Q0uTNeYjQ-ikvLAesJCexw](https://github.com/user-attachments/assets/941db480-f250-47ca-9935-50ac18b02ec1)

**Comment by yogesh0509** [comment]
Nice, these look really good. I cannot assign you the project before 25th but feel free to clone the repo and look around. Are you familiar with the front-end tech stack for this project. (tailwind CSS and wagmi)?

**Comment by sanaa9012** [comment]
Sure. I will be glad if you assign this task to me.  Yes, I am familiar with tailwind CSS and also have worked with it, wagmi I have started to learn.

**Comment by yogesh0509** [comment]
Cool, I will assign the issue to you after 25th. One important thing the current front-end of the project is not visible as the current contracts are deployed on polygon Mumbai which are deprecated. So either you have to wait until this [issue] (https://github.com/yogesh0509/web3-DREAM11/issues/10) is merged or you can find another way to temporarily disable metamask firewall in your local to view the current UI. To understand current UI is important as it will give you a idea of all the current features in our project.

**Comment by sanaa9012** [comment]
Yes, sure. I will start working with either of these alternatives.

**Comment by yogesh0509** [comment]
@sanaa9012 I have assigned you the issue. Do let me know if the issue is too big and you want me to break it down into smaller issues.

**Comment by sanaa9012** [comment]
@yogesh0509 Sure. Please break it down into smaller issues, easier for me to finish fast.

**Comment by yogesh0509** [comment]
Sure, can you provide a list of components from the current frontend that would be refactored along with new pages that you intend to add?
I will create separate issues. Also tell which page or component are you working on first?

**Comment by yogesh0509** [comment]
@sanaa9012 any updates??

**Comment by sanaa9012** [comment]
@yogesh0509 not yet. Working on it.

**Comment by yogesh0509** [comment]
> Sure, can you provide a list of components from the current frontend that would be refactored along with new pages that you intend to add? I will create separate issues. Also tell which page or component are you working on first?

@sanaa9012 If possible can you do this first. I will create those issues which will make it easy for you to work on issues. Also can you post updates once a week on discord server, so that it is easier for other devs to communicate and track progress.

**Comment by sanaa9012** [comment]
@yogesh0509 i won't be able to contribute for a week from now due to severe health issues. i will be glad if you understand the situation and give me some more time. please.

**Comment by yogesh0509** [comment]
@sanaa9012 It's alright. Don't worry!! 😁

**Comment by yogesh0509** [comment]
Hello @sanaa9012 , are you working on this issue currently?

**Comment by sanaa9012** [comment]
@yogesh0509 No i am not. Occupied. sorry
